### PR TITLE
Change target directory of generated code

### PR DESCRIPTION
**Pull request by holzeis**
@klochowicz I am still in favor of this for the reason as described in 77f66274da07d4cd2f556e73a53f9924fa902153

### CODE REVIEW

**Review by klochowicz — COMMENTED** [review]
Personally, I prefer keeping the current structure as the added clippy warning is weird and the default is the spot when everyone who would like to start contributing would look for it (after reading the frb tutorial).
that said, I can live with this if others are happy with it and it makes a difference for you :-) 

as an alternative: you can probably exclude the generated code in the project configuration (for me in emacs it doesn't come up at all, because of being present in gitignore)

**Review by da-kami — APPROVED** [review]
I'm in favor for this because one of the initially most confusing things when understanding the project structure was distinguishing generated code from code that I can edit.
This change helps with that as far as I understand :)

**Comment by klochowicz** [comment]
> I'm in favor for this because one of the initially most confusing things when understanding the project structure was distinguishing generated code from code that I can edit. This change helps with that as far as I understand :)

cool, let's merge it then!

File: .gitignore [hunk 1]
```diff
@@ -6,5 +6,6 @@
 .flutter-plugins*
 /build/
 # generated files by flutter-rust-bridge
-**/bridge_generated*
-**/bridge_definitions.dart
\ No newline at end of file
+bridge_generated.h
+**/bridge_generated/
+!/rust/src/bridge_generated/mod.rs
```
[the remark this answers is not in the document]
Comment (holzeis) [reply .gitignore:11]: this is an exclusion as the mod.rs does not get generated. line 10 does cover it but, I don't want git to ignore that file.

**Pull request merged**